Source/FolioReaderContainer.swift

```diff
@@ -38,31 +38,19 @@ public class FolioReaderContainer: UIViewController {
         epubPath = path
         shouldRemoveEpub = removeEpub
         super.init(nibName: nil, bundle: NSBundle.frameworkBundle())
-        
-        // Init with empty book
-        book = FRBook()
-        
-        // Register custom fonts
-        FontBlaster.blast(NSBundle.frameworkBundle())
-        
-        // Register initial defaults
-        FolioReader.defaults.registerDefaults([
-            kCurrentFontFamily: 0,
-            kNightMode: false,
-            kCurrentFontSize: 2,
-            kCurrentAudioRate: 1,
-            kCurrentHighlightStyle: 0,
-            kCurrentTOCMenu: 0,
-            kCurrentMediaOverlayStyle: MediaOverlayStyle.Default.rawValue,
-            kCurrentScrollDirection: FolioReaderScrollDirection.vertical.rawValue
-        ])
 		FolioReader.sharedInstance.readerContainer = self
-
-		readerConfig.canChangeScrollDirection = isDirection(readerConfig.canChangeScrollDirection, readerConfig.canChangeScrollDirection, false)
+		FolioReaderContainer.initSetup()
     }
     
     required public init?(coder aDecoder: NSCoder) {
-        fatalError("storyboards are incompatible with truth and beauty")
+		FolioReaderContainer.initSetup()
+
+		// Default values when the init called from storyboard
+		// TODO: Integrate contraints or storyboard to be more dynamic https://github.com/FolioReader/FolioReaderKit/issues/119
```

Comment:
No it's a future todo, I think it would be cleaner if we will use constraints instead of hardcoded values.